File: app/src/main/java/com/main/climbingdiary/models/Styles.java
package com.main.climbingdiary.models;

import java.util.Arrays;

public class Styles {
    public static String[] getStyle(Boolean UpperCase){
        if(UpperCase){
            return new String[]{"OS","RP","FLASH"};
        }else{
            return new String[]{"os","rp","flash"};
        }
    }

    public static int getStyleRatingFactor(String style){
        return (Arrays.asList(getStyle(false)).indexOf(style.toLowerCase())*5);
    }
}

File: app/src/main/java/com/main/climbingdiary/models/Styles.kt
package com.main.climbingdiary.models

import java.util.*

object Styles {

    fun getRP(lowercase: Boolean = false): String {
        return if (lowercase) "rp" else "RP"
    }

    fun getOS(lowercase: Boolean = false): String {
        return if (lowercase) "os" else "OS"
    }

    fun getFLASH(lowercase: Boolean = false): String {
        return if (lowercase) "flash" else "FLASH"
    }

    fun getStyle(UpperCase: Boolean): Array<String> {
        return if (UpperCase) {
            arrayOf(getOS(), getRP(), getFLASH())
        } else {
            arrayOf(getOS(true), getRP(true), getFLASH(true))
        }
    }

    fun getStyleRatingFactor(style: String): Int {
        return listOf(*getStyle(false))
            .indexOf(style.toLowerCase(Locale.ROOT)) * 5
    }
}